### ISSUE TITLE
508-defect-3 [SCREENREADER]: STEM Standalone - <legend> element is not anncouncing for screen readers

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#18250]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

Noah Gelman

**VFS Point of Contact:** _First name only. Delete if not applicable._

## User Story or Problem Statement

In several sections of the form, where the `<fieldset>` and `<legend>` elements are used, the content of the `<legend>` field are not being announced. The `<legend>` should be announced when navigating the form fields inside the `<fieldset>`

## Details

Screen reader announcement for `<legend>` field should announce when the form field inside `<fieldset>` is focused and hovered.

## Acceptance Criteria

- [ ] Make the `<legend>` element a direct descendent of the `<fieldset>` element to make it announce correctly

## Steps to Recreate

https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/applicant/information

Navigate the form as normal. Various sections use the `<fieldset>` and `<legend>` elements

**Comment by desireeturner** [comment]
@noahgelman it looks like there are some formatting issues with the description of this issue. Could you take a look and clarify? Thank you!

**Comment by ksantiagomorgan6** [comment]
This looks like a form systems issue, I was tracking it to `src/platform/forms-system/src/js/components/FieldTemplate.jsx`. Our code doesn't generate the `<legend>` or `<fieldset>` tags.

**Comment by desireeturner** [comment]
@noahgelman Are you able to pass this along to the right group to address this form system issue? We do not normally address those. Thank you!

**Comment by af-manish** [comment]
@noahgelman <cc @ERRonat0609 @mrblanco1 > gentle nudge/thoughts here?

**Comment by ERRonat0609** [comment]
@af-manish @mrblanco1 In reviewing this ticket, it appears that it was passed to Platform for review back in January because it was a form system issue. It was assigned back to @noahgelman on Jan 15 for follow-up.

**Comment by noahgelman** [comment]
@af-manish @desireeturner  @ERRonat0609 This issue has been resolved since posting this ticket

**Comment by ERRonat0609** [comment]
Based on the comment above, this issue has been resolved and can be closed.
